tests/csm/rest/test_query_deployment.py

```diff
@@ -42,14 +45,23 @@ def setup_class(cls):
         cls.ha_obj = HAK8s()
         cls.deploy_lc_obj = ProvDeployK8sCortxLib()
         cls.csm_obj = csm_api_factory("rest")
-        cls.csm_conf = configmanager.get_config_wrapper(fpath="config/csm/test_rest_capacity.yaml")
+        cls.csm_conf = configmanager.get_config_wrapper(
+            fpath="config/csm/test_rest_query_deployment.yaml")
         cls.update_seconds = cls.csm_conf["update_seconds"]
+        cls.failed_pod = None
+        cls.kvalue = None
+        cls.deploy_list = cls.csm_obj.master.get_deployment_name(POD_NAME_PREFIX)
+        cls.random_string = ''.join(random.choices(string.ascii_letters + string.digits, k=10))
+        cls.random_number = cls.csm_obj.random_gen.randrange(1, 99999)
+        cls.random_symbols = ''.join(random.choices(string.punctuation, k=10))
 
     def setup_method(self):
         """
         Setup method
         """
         self.log.info("Prerequisite: Deploy cortx cluster")
+        deploy_start_time = time.time()
```

Comment:
Is this required?

Reply:
it is required to Verify deployment time between start and end deployment time in some tests hence added.

Reply:
I dont see it getting used now but if u need for later tests also, It should be self.deploy_start_time and not just deploy_start_time

Reply:
ok noted

Reply:
done